aptos-move/e2e-move-tests/src/tests/transaction_context.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright © Aptos Foundation
+
+use crate::{
+    assert_success,
+    tests::common,
+    transaction_context::{create_many_uuids, initialize},
+    MoveHarness,
+};
+use aptos_language_e2e_tests::account::Account;
+
+fn setup() -> (MoveHarness, Account) {
+    initialize(common::test_dir_path("transaction_context.data"))
+}
+
+#[test]
```

Comment:
Can you add a test where an object is created and check that you can fetch resources there?

Reply:
Still working on it. Not entirely sure how to interact with objects.

Reply:
You can use the same /resources API to fetch resources at an object address. As for how to get the object address, this would be a good exercise for you to understand how people would be able to get the created addresses from the newly introduced functions.